tipg/dbmodel.py

```diff
@@ -1048,11 +1049,26 @@ async def get_collection_index(
                         break
 
             # Datetime Column
-            datetime_columns = [
-                c
-                for c in table.get("datetime_columns", [])
-                if c["name"] in property_names
-            ]
+            datetime_columns = []
+            datetime_column = None
+            for c in table.get("datetime_columns", []):
+                print(c, property_names)
+                if c["name"] in property_names:
+                    # get min/max from database
+                    mindt, maxdt = await conn.fetchrow_b(
```

Comment:
What do you think about the Settings to enable/disable this?

Reply:
I think that would be reasonable. I would leave it on as the default.